libraries/aws/provisioning/include/types/aws_iot_provisioning_types.h

```diff
@@ -277,6 +277,41 @@ typedef struct AwsIotProvisioningRejectedResponse
     size_t errorMessageLength;  /**< The length of the error messsage sent by the server. */
 } AwsIotProvisioningRejectedResponse_t;
 
+/**
+ * @ingroup provisioning_datatypes_paramstructs
+ * @brief Aggregates the data sent as response from AWS IoT Core for the request to create a new certificate for a CSR request.
```

Comment:
Re-word: `for` -> `to` and delete last `request`. E.g. `Aggregates the data sent as AWS IoT Core's response to the request to create  a new certificate for a CSR.` or even `Aggregates the data sent as AWS IoT Core's response to the CSR certificate creation request.`

Reply:
Updated with `Aggregates the data from AWS IoT Core's response to the request of a new certificate with a CSR.`